talk/talk-api.yaml

```
@@ -1770,13 +1771,36 @@ paths:
                 $ref: '#/components/schemas/TenantSettings'
         default:
           $ref: '#/components/responses/ErrorResponse'
+  /tenant:
```

```suggestion
  /tenants:
```
all endpoints are plural too